static/src/stylesheets/inline/article-labour-liverpool.scss

```diff
@@ -28,11 +28,65 @@ $labour-liverpool-header-element-opacity: .85;
 .content--labour-liverpool .content__article-body > p:first-of-type {
     @include fs-headline(3);
     color: darken($news-support-4, 25%);
-    margin-top: $gs-baseline;
     font-size: 22px;
     font-weight: 200;
 }
 
+.content--labour-liverpool .content__article-body {
+    padding-top: 12px;
+}
+
+.content--labour-liverpool.heroic--video .content__article-body {
+    padding: 0px;
+    @include mq($from: tablet) {
+        padding-top: 40px;
+    }
+}
+
+.content--labour-liverpool-article {
+    &.heroic--video {
+        min-height: 0rem;
+
+        @include mq($from: mobileLandscape) {
+            min-height: 27rem;
```

Comment:
Would it be possible to convert these values to `px` rather than `rem`, just for consistency?